source/guide_gitea.rst

```diff
@@ -96,93 +157,276 @@ and a new config file at ``~/gitea/custom/conf/app.ini`` to specify the port and
   [service]
   DISABLE_REGISTRATION = true
 
-Gitea provides many other configuration options. Take a look at the Documentation_
-to learn more.
+When using this, we have to finish the installation via gitea web service https://isabell.uber.space/install. This is exposed without any password request. We improve the configiguration with some modifications, e.g.:
+
+* Filling the database access data that we would otherwise enter in the web installation step. (``[database]`` section)
+* As security feature we lock the web installation and change the default password complexity to allow well to remember and secure passwords. (See `XKCD No. 936 <https://xkcd.com/936/>`_  and `Explained XKCD No. 936 <https://explainxkcd.com/wiki/index.php/936:_Password_Strength>`_ for the math behind it. 😉 ``[security]`` section)
+* We disallow public registration and set some privacy settings. (``[service]`` section)
+* Naming the repositories path.
+* Set the eye-friendly dark theme as default theme, set mailing, set time and set logging options.
+
+For more informations about the possibilities and configuration options see the Gitea documentation_ and the Gitea `configuration sample <https://github.com/go-gitea/gitea/blob/master/custom/conf/app.ini.sample>`_.
 
-Configure web server
---------------------
+.. note:: The minimum improvements are written in the sections of ``[database]``, ``[security]`` and ``[service]``.
 
-.. note::
+.. warning:: Replace ``isabell`` with your username, fill the database password ``PASSWD =`` with yours and enter the generated random into ``SECRET_KEY =``.
 
-    Gitea is running on port 9000.
+.. code-block:: ini
 
-.. include:: includes/web-backend.rst
+  APP_NAME = Gitea
+  RUN_USER = isabell
+  RUN_MODE = prod ; Either "dev", "prod" or "test", default is "dev".
+  
+  [server]
+  HTTP_PORT            = 9000
```

Comment:
Feel free to omit this and switch to the default port 9000 to save a line of config.

Reply:
It's already port 9000. I'm sorry, I don't see it.

Reply:
yes, 9000 is the default so you can just omit that line entirely.

Reply:
I copied it from the last version of this *.ini.

```
[server]
HTTP_PORT = 9000
DOMAIN = isabell.uber.space
ROOT_URL = https://%(DOMAIN)s

[service]
DISABLE_REGISTRATION = true
```

Reply:
They changed the default port to 3000. Should we change this to, or stay at 9000?

-- https://docs.gitea.io/en-us/config-cheat-sheet/

Reply:
3000 sounds good!